packages/server/lib/browsers/webkit.ts

```diff
@@ -121,7 +121,7 @@ export async function open (browser: Browser, url: string, options: BrowserLaunc
      */
     private suppressUnhandledEconnreset () {
       unhandledExceptions.handle((err: NodeJS.ErrnoException) => {
-        return err.code === 'ECONNRESET'
+        return err.code !== 'ECONNRESET'
```

Comment:
What is this doing here? Should this have it's own issue to track it?

Reply:
Mistakenly included this in the diff, I'll open a separate PR for it.

```suggestion
        return err.code === 'ECONNRESET'
```